.github/workflows/verible-ci.yml

```diff
@@ -51,23 +53,38 @@ jobs:
             FOLDER: /tmp/pages
             CLEAN: true
 
+
+   Matrix:
+      runs-on: ubuntu-latest
+      outputs:
+        matrix: ${{ steps.generate.outputs.matrix }}
+      steps:
+      - uses: actions/checkout@v2
+      - id: generate
+        run: |
```

Comment:
I think this would be better as an external script rather than encoded in the `.github/workflows/verible-ci.yml`?

Reply:
IE It would be nice to be able to run the script locally to see what the output looks...

Reply:
Done.